conjur-api/ResourceMetadata.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Conjur
+{
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    public class ResourceMetadata
+    {
+        [DataMember (Name = "id")]
+        public string Id { get; set; }
```

Comment:
I wonder, can these just be public members instead of properties, since these classes are really just bags of plain old data?

Reply:
After running StyleCop it was suggest that members need to be private. Since those can't be - we will leave them as properties with getters & setters